pom.xml

```diff
@@ -7,7 +7,7 @@
     <version>${product.version}</version>
```

Comment:
We need to get rid of the variable here, since having a variable in the version is deprecated. Also the new `version.sh` script expects a constant.

That also means we need to reference the product version differently in dependent locations like the dist file name. Please look that up.